perfdash/perfdash.go

```diff
@@ -45,18 +46,26 @@ var (
 	globalConfig = make(map[string]string)
 )
 
-func initGoogleDownloaderOptions() {
+// GCSDownloaderOptions is an options for GoogleGCS specific Downloader.
+type GCSDownloaderOptions struct {
+	CredentialPath string
+}
+
+func initDownloaderOptions() {
 	pflag.BoolVar(&options.OverrideBuildCount, "force-builds", false, "Whether to enforce number of builds to process as passed via --builds flag. "+
 		"This would override values defined by \"perfDashBuildsCount\" label on prow job")
 	pflag.IntVar(&options.DefaultBuildsCount, "builds", maxBuilds, "Total builds number")
 	pflag.StringArrayVar(&options.ConfigPaths, "configPath", []string{}, "Paths/urls to the prow config")
 	pflag.StringArrayVar(&options.GithubConfigDirs, "githubConfigDir", []string{}, "Github API url to the prow config directory, all configs from this dir will be used."+
 		"To specify more than one dir, this arg shall be specified multiple times, one time for each dir.")
-	pflag.StringVar(&options.CredentialPath, "credentialPath", "", "Path to the gcs credential json")
 	pflag.StringVar(&options.LogsBucket, "logsBucket", "kubernetes-jenkins", "Name of the data bucket")
```

Comment:
Do you need LogsBucket or LogsPath in this anymore?

Reply:
Removed LogsBucket and LogsPath from here; moved it to be declared as a var